go/service/ui_router.go

```diff
@@ -155,3 +155,20 @@ func (u *UIRouter) GetRekeyUI() (keybase1.RekeyUIInterface, int, error) {
 	}
 	return ret, sessionID, nil
 }
+
+func (u *UIRouter) GetRekeyUINoSessionID() (keybase1.RekeyUIInterface, error) {
```

Comment:
our typical logic is if there's no sessionid, it was spawned by the daemon. if it has a sessionid its cause electron spawned it and it using it to bookkeep.